src/components/editMeasure/populationCriteria/populationCriteriaSideNav/PopulationCriteriaSideNav.tsx

```diff
@@ -74,6 +74,10 @@ export default function PopulationCriteriaSideNav(
   const initiateNavigateGroupClick = (e) => {
     e.preventDefault();
     const groupNumber = Number(e.target.id);
+    // edge case
+    if (pathname !== groupsBaseUrl) {
+      history.push(groupsBaseUrl);
```

Comment:
Would this work, if I want to navigate from base configuration to "Criteria other than 1"

Reply:
Sounds like it will, I just learned that not every criteria has its own unique Uri

Reply:
I got it to work on any. 
We don't preserve any state for groupNumber in the url so it defaults to whatever state is passed to it.

This really should be changed in the future to use url params. It would make everything much simpler.